FAIMS/FAIMS3
# Overview Map of Record locations

# Overview Map of Record locations

## JIRA Ticket

None (Fieldmark)

## Description

Adds a map tab to the notebook/survey view to show the locations stored in existing records for the notebook. 

## Proposed Changes

- Add the OverviewMap component to display GIS locations

## How to Test

View a notebook with existing records containing GIS fields (eg. Rapid Impact Survey), view the Map tab to see the record locations. 

Also look at a notebook with no GIS fields or no records to see a suitable message.


## Additional Information

This is the first version of this feature, many refinements are possible.

## Checklist
